### PR TITLE
RTIR - Added referer header parameter

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17615

## Description
Added new optional paremeter - Referer: Adds a referer header to the requests sent by the integration

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation
- [ ] Code Review

### CODE REVIEW

**Comment by Itay4** [comment]
dont really understand why would a user want to fill in the referer..
if it works with empty and covers the issue, lets just always pass it empty

**Comment by DeanArbel** [comment]
> dont really understand why would a user want to fill in the referer..
> if it works with empty and covers the issue, lets just always pass it empty

That's the thing, it shouldn't be empty - it's used by the RTIR system to prevent CSRF - basically if this protection is turned on, then it checks the referer header to see if the site is in its whitelist  (it's a pretty lame protection but that's how they work).

In the issue RTIR returned an error for not having the header, however they can also return an error if the header is in the request, but its value is not in the whitelist...

File: Integrations/RTIR/RTIR.yml [hunk 1]
```diff
@@ -569,3 +573,4 @@ script:
   type: python
 tests:
 - RTIR Test
+releaseNotes: "Added new optional paremeter - Referer: Adds a referer header to the requests sent by the integration."
```
Comment (Itay4) [inline Integrations/RTIR/RTIR.yml]: rn should be in a seperate file

**Comment by Itay4** [comment]
@kirbles19 please review the added parameter in the yml and the detailed instructions in the md file

**Comment by kirbles19** [comment]
@DeanArbel All files reviewed.

**Comment by Itay4** [comment]
@kirbles19 i think we should update the display name of the parameter, as it is not clear enough to me, from the current display name (Referrer), what this parameter is used for. wdyt?

File: Integrations/RTIR/RTIR.yml [hunk 2]
```diff
@@ -44,6 +44,10 @@ configuration:
   name: fetch_queue
   required: false
   type: 0
+- display: Referer
```
Comment (Itay4) [inline Integrations/RTIR/RTIR.yml]:
```suggestion
- display: Referer request header
```

Comment (DeanArbel) [reply Integrations/RTIR/RTIR.yml]: 👍

**Pull request merged**